pkg/prometheus/promcfg.go

```diff
@@ -4492,21 +4492,25 @@ func (cg *ConfigGenerator) appendOTLPConfig(cfg yaml.MapSlice) (yaml.MapSlice, e
 		return cfg, nil
 	}
 
-	if cg.version.LT(semver.MustParse("2.55.0")) {
-		return cfg, fmt.Errorf("OTLP configuration is only supported from Prometheus version 2.55.0")
```

Comment:
I'd prefer to keep the error: if users configure OTLP with Prometheus < v2.55, it's better to tell them that they need to upgrade.